charts/country-risk/country-risk-backend/templates/secret-pgadmin4.yaml

```diff
@@ -0,0 +1,30 @@
+################################################################################
+# Copyright (c) 2022,2023 BMW Group AG
+# Copyright (c) 2022,2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+################################################################################
+
+apiVersion: v1
+kind: Secret
+metadata:
+  annotations:
+    avp.kubernetes.io/path: {{ .Values.pgadmin4.secret.path }}
```

Comment:
> Not sure, what is happening here, but there are a lot of new unchecked 3rd party dependencies. I created the IP issues to check the 3rd party libraries. Please track them and update the `DEPENDENCIES` ones they are resolved. Please also try to avoid making this PR bigger and bigger. It is already at a size, that now requires a manual IP check for project content
> 
> **Project content review**: https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8427
> 
> **List of 3rd party library checks**:
> 
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8386
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8387
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8388
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8374
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8389
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8390
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8391
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8392
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8393
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8394
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8395
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8396
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8397
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8398
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8399
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8400
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8401
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8402
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8403
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8404
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8375
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8405
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8406
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8407
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8408
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8409
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8410
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8411
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8412
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8413
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8414
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8415
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8416
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8417
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8418
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8419
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8420
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8421
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8422
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8423
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8424
> * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8425

Thanks Sebastian , I updated and changed some libs that were being found with transient vulnerability's .  To be able to pass this security findings  https://jira.catena-x.net/browse/CXRM-1042 and since the parent lib didn't had new releases i needed to fixed the sub libs and upgrade them.

Reply:
> > Not sure, what is happening here, but there are a lot of new unchecked 3rd party dependencies. I created the IP issues to check the 3rd party libraries. Please track them and update the `DEPENDENCIES` ones they are resolved. Please also try to avoid making this PR bigger and bigger. It is already at a size, that now requires a manual IP check for project content
> > **Project content review**: https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8427
> > **List of 3rd party library checks**:
> > 
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8386
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8387
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8388
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8374
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8389
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8390
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8391
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8392
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8393
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8394
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8395
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8396
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8397
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8398
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8399
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8400
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8401
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8402
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8403
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8404
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8375
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8405
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8406
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8407
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8408
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8409
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8410
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8411
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8412
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8413
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8414
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8415
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8416
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8417
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8418
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8419
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8420
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8421
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8422
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8423
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8424
> > * https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8425
> 
> Thanks Sebastian , I updated and changed some libs that were being found with transient vulnerability's . To be able to pass this security findings https://jira.catena-x.net/browse/CXRM-1042 and since the parent lib didn't had new releases i needed to fixed the sub libs and upgrade them.

Hi @fabiodmota, thanks for the info. Can you please paste some context from the link you provided here, since it is not accessible for everyone. Updates are fine with me, but still we need to wait for the IP issue approvals

Reply:
Dependabot detected vulnerability on d3-color lib that is used by d3-zoom lib that is used by react simple maps.
My only used lib is react simple maps that was already on the latest version (3.0.0)
So in order to upgrade the d3-color I upgraded the parent manually by overriding d3-zom

![image](https://github.com/eclipse-tractusx/vas-country-risk-frontend/assets/107047355/0da09af7-eaad-4752-8375-8406dba10e92)

With my upgrade on that many libs that the d3-zom uses also updated , and fixed my issue like we can see on the attach.